support-rest/src/main/scala/com/gu/rest/WebServiceHelper.scala

```diff
@@ -104,7 +104,7 @@ trait WebServiceHelper[Error <: Throwable] {
       case "2xx" =>
         decode[A](responseBody).left.map { err =>
           decodeError(responseBody).right.getOrElse(
-            WebServiceHelperError[A](codeBody, s"failed to parse response: $err", err)
+            WebServiceHelperError[A](codeBody, s"failed to parse response. Error was: $err, Response was: $responseBody", err)
```

Comment:
Do you want to leave this in? Not sure what uses this helper, so we might be logging something secret (for example).

Reply:
It goes through the SafeLogger so will be fine, but I will take it out once this has deployed successfully. I'd like to leave it in for now though in case of further errors.